src/tests/ftest/util/soak_utils.py

```diff
@@ -430,6 +429,9 @@ def launch_vmd_identify_check(self, name, results, args):
                     if device['ctrlr']['led_state'] != "QUICK_BLINK":
                         failing_vmd.append([device['ctrlr']['pci_addr'], value['hosts']])
                         status = False
+    # reset leds to previous state
+    for uuid in uuids:
+        self.dmg_command.storage_led_identify(ids=uuid, reset=True)
```

Comment:
Would you rather reset all at once? I imagine it's faster if there are a lot
```suggestion
    # reset leds to previous state
    self.dmg_command.storage_led_identify(ids=','.join(uuids), reset=True)
```

Reply:
No since it is a harasser I wanted to do it one at a time.

Reply:
Makes sense :)